tests/system_tests/test_blueapi_system.py

```diff
@@ -338,6 +260,30 @@ def test_get_current_state_of_environment(client: BlueapiClient):
     assert client.get_environment() == EnvironmentResponse(initialized=True)
 
 
+@pytest.mark.xfail(
```

Comment:
Must: This test should be skipped rather than xafiled, xfail is for tests that fail all the time, not intermittently. It just has the effect of reversing the result (i.e. if the test sometimes passes the suite will fail)

Reply:
Worth noting I have a long term solution WIP here: https://github.com/DiamondLightSource/blueapi/pull/743